tools/topology/topology2/platform/intel/lnl.conf

```diff
@@ -0,0 +1,8 @@
+# LNL-specific variable definitions
+Define {
+	DMIC_DRIVER_VERSION		3
+	SSP_BLOB_VERSION		0x105
+	# Lower IO clock for the LNL FPGA.
+	DMIC_IO_CLK			19200000
```

Comment:
should we have a lnl-fpga config?

Reply:
either that or I can pass this during build time and just reused mtl.conf. What do you think?

Reply:
I don't have a strong opinion other than we need to avoid mixing hardwares...

Reply:
ok modified to pass the DMIC_IO_CLOCK while generating sof-lnl-nocodec-fpga.tplg